tests/test_modeling_flax_bert.py

```diff
@@ -105,7 +105,13 @@ def prepare_config_and_inputs_for_common(self):
 @require_flax
 class FlaxBertModelTest(FlaxModelTesterMixin, unittest.TestCase):
 
-    all_model_classes = (FlaxBertModel,) if is_flax_available() else ()
+    all_model_classes = (FlaxBertModel, FlaxBertForMaskedLM) if is_flax_available() else ()
```

Comment:
all tests pass for `FlaxBertForMaskedLM` as well now => models are equal in PT and Flax